src/components/pages/LandingPage/LandingPage.less

```diff
@@ -0,0 +1,140 @@
+* {
+  box-sizing: border-box;
+}
+/*Section 1*/
+.landInfo1 {
+  display: flex;
+  padding-bottom: 5%;
+}
+.head1 {
+  padding-bottom: 10px;
+  font-size: 2.7rem;
+  font-weight: 700;
+  color: #003d71;
+}
+.left1 {
+  width: 50%;
+  padding: 5% 5% 0 5%;
+}
+.right1 {
+  width: 50%;
+}
+.text1 {
+  font-size: 1.125rem;
+  font-weight: 400;
+}
+.img1 {
+  width: 90%;
+  height: auto;
+  padding: 0 5% 0 5%;
+  margin-top: 80px;
+}
+
+/* content */
+.content-container {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  margin-top: 30px;
+}
+
```

Comment:
I like that they are broken up in sections. 

There seem to be spaces in between these styles, but no spaces in Section 1 - a format clean up might be nice to keep it consistent.